src/pages/Verification/Confirm.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Typography, CircularProgress } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import axios, { AxiosResponse } from 'axios';
+import { BASE_URL } from '@/config/default';
+import { selectToken } from "@/redux/userSlice";
+import { useSelector } from "react-redux";
+import { RootState } from '@/redux/store';
+
+
+interface AuthResponse {
```

Comment:
Why did you declare a new AuthResponse type? I already have one in the `src/types/AuthTypes.ts` file in case you didn't see it?